### DISCUSSION TITLE
Nobility titles

### DISCUSSION THREAD

**Discussion by fisharebest** [fisharebest/gedcom-name#6] [General]
GEDCOM provides `g7:INDI-TITL`.  For example:

```
1 NAME Charles Philip Arthur George
1 TITL King Charles III
```

Is there an overlap between `NAME` and `TITL`?

**Comment by Norwegian-Sardines** [comment]
> Is there an overlap between `NAME` and `TITL`?

Having monitored several different genealogy/family historian software "help forums", I've witnessed historians ignoring the `TITL` tag (entering "King Charles III" in the `NAME` tag) or the software not including `TITL` in identity references.

While GEDCOM can't stop this occurrence (including a title in the `NAME` tag), nobility titles and earned titles (Doctor, Mayor, General, etc) provide an important identifier in reports, diagrams, and sorts. And since NAME.[Subtag] are optional, in many cases the transmission may only contain the `NAME` tag or the receiving application may only process the `NAME` tag, therefore the title would/could become parsed as a given name and used incorrectly in later reports or other instances.

So, yes I believe that an overlap is present in these two tags!

**Reply by Tim-Partridge** [reply]
In the example you give I would regard King as a title associated with a name Charles. Monarchs in the UK adopt a Regnal Name on ascending the throne. This may or may not be the same as any of their previous names. Popes of the Catholic church do something similar. The number afterwards is more for convenience for historians in distinguishing people. Queen Elizabeth II signed her name Elizabeth R (R short for Regina, Latin for Queen).

**Comment by hartenthaler** [comment]
There are (at least) three types of titles:
- academic titles like "Dr.", or "PhD", or "Bachelor", or "Master", or "Dr. Dr. (hc)", ...
- professional or religious titles like "General", "Reverend", "Brother", or "Kammersänger" (there are still a few of those in Germany but many in Austria)
- Imperial, royal and noble ranks like "King of ...", "Prince", ....

Academic and professional titles can be used in front of a name like "Dr. Erich Maier" or "Mother Theresia" as a religious name. Today the title is normally not part of the name, but in former days the title could be part of the name (as a prefix). So in my opinion it would be **wrong** to have such titles as name pieces like 1 NAME Dr. Erich /Maier/. Such titles should be separate items; they can be used by a person after a specific date/event, and they are not part of the birth name. A person can collect several titles.

Noble titles are more complicated, some of them can be used instead of a name like "King Charles III".

Another example of a noble title: "[Otto Fürst von Bismarck](https://de.wikipedia.org/wiki/Otto_von_Bismarck)". I would say that this is his name when he died; the name pieces are: "Otto" = given name; "Fürst" = title; "von" = noble name prefix; "Bismarck" = surname. He was born in 1815 and got his noble title in 1871. And he had more noble titles like "Graf von Bismarck-Schönhausen" and "Herzog zu Lauenburg".

There is a law in Germany from 1918 that noble titles cannot be given to a person any longer. Persons with a noble title in 1918 changed their name and the noble title was from that time on a part of their name (so if there would be someone born in 2022 with the name "Otto Freiherr von Münchhausen", then "Otto" is the given name and "Freiherr von Münchhausen" is the surname). In Austria, the noble titles and the noble prefix "von" disappeared completely.

I would store him as

> 1 NAME Otto /Freiherr von Münchhausen/
> 2 GIVN Otto
> 2 SURN Freiherr von Münchhausen

But I would store the historic person "Hieronymus Carl Friedrich Freiherr von Münchhausen" (born 1720) as

> 1 TITL Freiherr von Münchhausen
> 1 NAME Hieronymus Carl Friedrich /von Münchhausen/
> 2 GIVN Hieronymus Carl Friedrich
> 2 SPFX von
> 2 SURN Münchhausen
> 2 NICK Lügenbaron


### Summary

Academic and professional "titles" are disjunct from "name". I would prefer to have TITL as an element in the INDIVIDUAL_ATTRIBUTE_STRUCTURE still available in the GEDCOM standard. The definition of TITL in the 7.0 standard says

> A formal designation used by an individual in connection with positions of royalty or other social status, such as Grand Duke.

This definition should be modified to make clear that academic and professional titles should be stored in this tag, too.

Noble titles are used sometimes in the middle of a name or they are used instead of a name. Noble titles should be stored in the TITL tag. How they are presented together with the name of a person is not part of the standard, it is an implementation issue.

**Reply by Tim-Partridge** [reply]
There are many personal, academic, religious, military, organisational and state granted titles. They normally have a particular order in which they are presented, depending on the culture. I think it would be helpful to be able to specify this postioning so that the name can be presented in the same manner as it appears in the source material. Asking implementations to know how and where every title in the world should appear is quite a burden. There are style guides available in some places, but there is no certainty that an individual or record follows the current fashion.

Titles can have spaces within them.  "Private 1st class"

A person may have many title components they are entitiled to use, but only use all of them in very formal situations. They may also be selective in which ones they use. If writing to their professional body they are likely to the title representing their ranking in that body.

**Comment by Norwegian-Sardines** [comment]
Names are identifiers if an individual, used very often to separate one person from another, very much like Jr., Sr. and other numbered name suffixes as well as professional suffixes like Esq.,  MD., and other post-nominal letters. These suffixes are generally time based as well or not really part of a name either yet are included as a both a `NAME_STRUCTURE` and in the `NAME` tag as part of an individual’s identity.  Names should also have a date range associated with them so that we can be used to explain the progression of name changes (including title and other identity based changes) over the life of a person.

I believe that based on family historians view of recording individual that includes professional, academic and noble titles that we should move our definition of how `NAME` is used to a `NAME` tag that shows a person’s identity rather than just their name.  This view aligns well with other proposed changes to include patronymic, location, gender, clan, and other non-western based individual identities.

> Academic and professional "titles" are disjunct from "name". I would prefer to have TITL as an element in the INDIVIDUAL_ATTRIBUTE_STRUCTURE still available in the GEDCOM standard.

While I agree that the `TITL` should be augmented to include additional criteria, it is never used as part of an individual’s identity in any modern software, which is wrong!  It should moved up to be part of the `NAME` Structure to correspond to actual usage by family historians.  Otherwise, many Kings and other nobility would just be displayed by a singular first name.

**Comment by Tim-Partridge** [comment]
Titles can be gained and lost / discarded over time.
For instance professional membership may confer one of several titles, which change as the person progresses. If the person leaves the organisation, the title is lost.

In the UK, doctors who progress to become surgeons traditionally drop the title of Dr and revert back to their original title. E.g. Dr Smith goes back to being Mrs Smith.

**Reply by Norwegian-Sardines** [reply]
> Titles can be gained and lost / discarded over time.
For instance professional membership may confer one of several titles, which change as the person progresses. If the person leaves the organisation, the title is lost.

Yes this is true, which is why I think using multiple `NAME` tags along with a `DATE` range for use is important for family historians.
